scripts/integration_config.yml.tpl

```diff
@@ -0,0 +1,5 @@
+---
+vcenter_hostname: ${VCENTER_HOSTNAME}
```

Comment:
This is a yml file I don't think it should be in script dir

Reply:
fixed, I moved them to be under integration folder and without the scripts folder

Reply:
thanks